app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgGraphBuilder.java

```diff
@@ -183,27 +183,41 @@ private void addBgReadingValues() {
         }
     }
 
-    public Line highLine() {
+    public Line highLine(){ return highLine(true);}
+
+    public Line highLine(boolean show) {
         List<PointValue> highLineValues = new ArrayList<PointValue>();
         highLineValues.add(new PointValue((float) start_time, (float) highMark));
         highLineValues.add(new PointValue((float) end_time, (float) highMark));
         Line highLine = new Line(highLineValues);
         highLine.setHasPoints(false);
         highLine.setStrokeWidth(1);
-        highLine.setColor(ChartUtils.COLOR_ORANGE);
+        if(show) {
+            highLine.setColor(ChartUtils.COLOR_ORANGE);
+        } else {
+            highLine.setColor(Color.TRANSPARENT);
+        }
         return highLine;
     }
 
-    public Line lowLine() {
+    public Line lowLine(){ return lowLine(true, false);}
+
```

Comment:
Point taken.  I'll update it at lunch time.
Also the comments, I meant to get rid of them in any case.
Cheers

On Tue, Jan 19, 2016 at 10:03 AM, Matthias Granberry <
notifications@github.com> wrote:

> In
> app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgGraphBuilder.java
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/254#discussion_r50053943>
> :
>
> >          return highLine;
> >      }
> >
> > -    public Line lowLine() {
> > +    public Line lowLine(){ return lowLine(true, false);}
> > +
>
> Using multiple booleans is an anti-pattern. It is better to add a static
> final const value named something like LINE_VISIBLE or LINE_HIDDEN to make
> calling code more readable.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/254/files#r50053943>
> .
>



-- 
John Stevens
"You are how you live, not what you have."